cmd.go

```diff
@@ -373,35 +367,33 @@ func handlePrint() (err error) {
 	default:
 		err = nil
 	}
-
 	return err
 }
 
-func handleYay() (err error) {
+func handleYay() error {
 	//_, options, targets := cmdArgs.formatArgs()
 	if cmdArgs.existsArg("gendb") {
-		err = createDevelDB()
-	} else if cmdArgs.existsDouble("c") {
-		err = cleanDependencies(true)
-	} else if cmdArgs.existsArg("c", "clean") {
-		err = cleanDependencies(false)
-	} else if len(cmdArgs.targets) > 0 {
-		err = handleYogurt()
+		return createDevelDB()
 	}
-
-	return
+	if cmdArgs.existsDouble("c") {
+		return cleanDependencies(true)
+	}
+	if cmdArgs.existsArg("c", "clean") {
+		return cleanDependencies(false)
+	}
+	if len(cmdArgs.targets) > 0 {
+		return handleYogurt()
+	}
+	return nil
 }
 
-func handleGetpkgbuild() (err error) {
-	err = getPkgbuilds(cmdArgs.targets)
-	return
+func handleGetpkgbuild() error {
+	return getPkgbuilds(cmdArgs.targets)
 }
 
-func handleYogurt() (err error) {
+func handleYogurt() error {
 	config.SearchMode = NumberMenu
-	err = numberMenu(cmdArgs.targets)
-
-	return
+	return numberMenu(cmdArgs.targets)
 }
 
 func handleSync() (err error) {
```

Comment:
If these changes are made, it is not needed to name the return

Reply:
Agree, I can update the pull request.